pkg/asset/installconfig/aws/availabilityzones.go

```diff
@@ -71,33 +71,35 @@ func describeAvailabilityZones(ctx context.Context, session *session.Session, re
 
 // zonesByType retrieves a list of zones by a given ZoneType attribute within the region.
 // ZoneType can be availability-zone, local-zone or wavelength-zone.
-func zonesByType(ctx context.Context, session *session.Session, region string, zoneType string) ([]string, error) {
+func zonesByType(ctx context.Context, session *session.Session, region string, zoneTypes []string) ([]string, error) {
```

Comment:
What about keeping this function for only one type and then calling it multiple times, one for each zone type?

Reply:
Or maybe move the `describeAvailabilityZones` out of this function, and make it a `filterZonesByType(zones []*ec2.Zone, zoneType string)`

Reply:
Good suggestion! Changed 👍🏽